src/java.base/unix/native/libjli/java_md_solinux.c

```diff
@@ -402,7 +425,7 @@ CreateExecutionEnvironment(int *pargc, char ***pargv,
                     /* On AIX we additionally need 'jli' in the path because ld doesn't support $ORIGIN. */
                     JLI_StrLen(jrepath) + JLI_StrLen("/lib//jli:") +
                     /* On AIX P9 or newer with NX accelerator enabled, add the accelerated zlibNX to LIBPATH */
-                    ((power_9_andup() && power_nx_gzip()) ? JLI_StrLen(":" ZLIBNX_PATH) : 0) +
```

Comment:
While you're here, can you remove the extra `/` on line 426? It's leftover from changes in jdk8 where there's the extra "arch" directory.

Reply:
Updated.